docs/api/routing.rst

```diff
@@ -374,6 +374,60 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other architectures. Consider
+the task of building an API for a calculator which can both add and subtract two numbers. You could implement the 
+following:
+
+.. code:: python
+
+    class Add():
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) + int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    class Subtract(object):
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) - int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    add = Add()
+    subtract = Subtract()
+    api = falcon.API()
+    api.add_route('/add', add)
+    api.add_route('/subtract', subtract)
+
+However, in addition to being slightly verbose it's also logically a bit strange because adding and
```

Comment:
I'm not a native speaker and I'm particularly bad at punctuation, but [with that disclaimer in mind] I would put a comma:
> However, in addition to being slightly verbose, it's also logically a bit strange <...>

Reply:
I think you're right (I think it's a non-defining clause?). Updated